yarn.lock

```diff
@@ -385,7 +385,7 @@
     chalk "^2.0.0"
     js-tokens "^4.0.0"
 
-"@babel/parser@^7.0.0", "@babel/parser@^7.4.4", "@babel/parser@^7.8.0", "@babel/parser@^7.8.6":
+"@babel/parser@^7.0.0", "@babel/parser@^7.4.4", "@babel/parser@^7.8.0":
```

Comment:
Was yarn.lock out of sync?

Reply:
Looks like https://github.com/parcel-bundler/parcel/pull/4935/files didn't update it...